test/integration/bodyDictionary.spec.ts

```diff
@@ -0,0 +1,496 @@
+import { assert } from "chai";
```

Comment:
Now I feel bad because I used `expect` on my last tests (I think I copied that from another test). Do you prefer `assert`? Happy to use that going forward.

Reply:
I had been using expect, but started using assert in this one since it looks like it would be the approach will be taking in the SDKs repo. I don't have a strong opinion about it, but if we'll be using assert in the SDKs it would probably be easier for everyone to use it as well here. I'll have to go back and convert other tests to assert